index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+    <header>
+    <div class="pure-nav-list">
+            <h4 class="important"><a href="#">Ayodele</a></h4>
+            <nav>
+                <div class="menu1">
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                    <div class="menu"></div>
+                </div>
+            </nav>
+            </div>
+            
+        </header>
```

Comment:
### OPTIONAL

- [ ] The header is not like in the original design. You can change it to make it better. (ignore the yellow border in the picture.)
![image](https://user-images.githubusercontent.com/38635158/143505182-2198ffe8-a0cb-45af-8b41-7308cf0845e9.png)

___
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/143505109-838cecf0-8f05-44f3-8d89-53d6a096aed8.png)